src/app/zap_cluster_list.json

```diff
@@ -37,6 +37,7 @@
         "DISHWASHER_MODE_CLUSTER": [],
         "MICROWAVE_OVEN_MODE_CLUSTER": [],
         "DOOR_LOCK_CLUSTER": [],
+        "ECOSYSTEM_INFORMATION_CLUSTER": [],
```

Comment:
We also need to add this cluster to  "ServerDirectories" in line 187

Reply:
Thanks!